### PR TITLE
flow: ovs mirror probe

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

**Comment by safchain** [comment]
Waiting for the ovsdb metrics patch to be merged

**Comment by safchain** [comment]
rerun functional-tests-backend-orientdb

File: flow/probes/ovsmirror.go [hunk 1]
```diff
@@ -0,0 +1,592 @@
...
+
+	extIds := make(map[string]string)
+	extIds["skydive-probe-id"] = probe.id
+	ovsMap, err := libovsdb.NewOvsMap(extIds)
+	if err != nil {
+		return nil, err
+	}
+	intfRow["external_ids"] = ovsMap
+
+	insertOp := libovsdb.Operation{
```
Comment (lebauce) [inline flow/probes/ovsmirror.go]: one line for the win

File: flow/probes/ovsmirror.go [hunk 2]
```diff
@@ -0,0 +1,592 @@
...
+
+	extIds := make(map[string]string)
+	extIds["skydive-probe-id"] = probe.id
+	ovsMap, err := libovsdb.NewOvsMap(extIds)
+	if err != nil {
+		return nil, err
+	}
+	portRow["external_ids"] = ovsMap
+
+	insertOp := libovsdb.Operation{
```
Comment (lebauce) [inline flow/probes/ovsmirror.go]: one line for the win

File: flow/probes/ovsmirror.go [hunk 3]
```diff
@@ -0,0 +1,592 @@
...
+
+	extIds := make(map[string]string)
+	extIds["skydive-probe-id"] = probe.id
+	ovsMap, err := libovsdb.NewOvsMap(extIds)
+	if err != nil {
+		return nil, err
+	}
+	mirrorRow["external_ids"] = ovsMap
+
+	insertOp := libovsdb.Operation{
```
Comment (lebauce) [inline flow/probes/ovsmirror.go]: one line for the win

**Comment by safchain** [comment]
rerun functional-tests-backend-elasticsearch

**Comment by safchain** [comment]
rerun functional-tests-backend-orientdb

**Pull request merged**